cloudygames/models.py

```diff
@@ -8,15 +18,68 @@ class Game(models.Model):
     address = models.CharField(max_length=45)
     users = models.ManyToManyField(User)
 
+class GameSession(models.Model):
+    player = models.ForeignKey(User)
+    game = models.ForeignKey(Game)
+    controller = models.IntegerField()
+
+    def joinGame(self, _game):
```

Comment:
As the creation of the GameSession is inside the view, I believe join_game would not need the return value of a game session. I apologize if this is incorrect though.

Can you explain more about GameSessionFullException?